azure-monitor-opentelemetry/samples/logging/custom_properties.py

```diff
@@ -21,3 +21,5 @@
 
 # Pass custom properties in a dictionary with the extra argument
 logger.debug("DEBUG: Debug with properties", extra={"debug": "true"})
+
+input()
```

Comment:
Is there a reason for adding input to all the log samples?

Reply:
It's to add delay to the python app so it doesn't close before exporting.